modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,6 +1,100 @@
 = Release Notes for Couchbase Server 7.1
 :description: pass:q[Couchbase Server 7.1 introduces multiple new features.]
 
+[#release-711]
+== Release 7.1.1 (July 2022)
+
+Couchbase Server 7.1.1 was released in July 2022.
+This maintenance release contains fixes to issues.
+
+=== Fixed Issues
+
+This release contains the fixes listed below.
+
+=== Data Service
+
+[#table-known-issues-711-dataservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52248[MB-52248^]
+| Memcached hangs when no password is passed for encrypted private key
```

Comment:
The terminology used in the docs etc for encrypted private keys, is that they are unlocked with a passphrase not a password. Ref: https://docs.couchbase.com/server/current/rest-api/upload-retrieve-node-cert.html#json-passphrase-registration